project/src/components/page-footer/page-footer.tsx

```diff
@@ -1,9 +1,9 @@
+import Logo from '../logo/logo';
+
 export default function PageFooter() {
   return (
     <footer className="footer container">
-      <a className="footer__logo-link" href="main.html">
-        <img className="footer__logo" src="img/logo.svg" alt="6 cities logo" width="64" height="33" />
-      </a>
+      <Logo type='footer' />
```

Comment:
в jsx обычно двойные кавычки
```suggestion
      <Logo type="footer" />
```